src/scene/axis.js

```diff
@@ -330,7 +330,7 @@ function vg_axisLabelExtend(orient, labels, oldScale, newScale, size, pad) {
   if (orient === "top" || orient === "bottom") {
     vg.extend(labels.properties.enter, {
       x: oldScale,
-      y: {value: size},
+      y: {value: size}
```

Comment:
i would rather not enforce es3 - trailing commas are good at minimizing diffs. Since this project won't work before Canvas support anyway, I think we should allow them in.